projects/demo/src/modules/experimental/tooltip/examples/2/index.html

```diff
@@ -0,0 +1,40 @@
+<ng-template #tooltip>Allowed symbols: ♠ ♣ ♦ ♥</ng-template>
+
+<tui-input
+    tuiTextfieldSize="m"
+    class="input"
+    [tuiTextfieldLabelOutside]="true"
+    [(ngModel)]="value"
+></tui-input>
+<tui-tooltip
+    direction="bottom"
+    [content]="tooltip"
+></tui-tooltip>
+
+<tui-input
+    tuiTextfieldSize="m"
+    class="input"
+    [tuiTextfieldLabelOutside]="true"
+    [(ngModel)]="value"
+></tui-input>
+<tui-tooltip
+    content="Set icon color with 'color'"
```

Comment:
I've not noticed any color's change 🤔 
<img width="514" alt="Screenshot 2023-11-20 at 16 06 04" src="https://github.com/taiga-family/taiga-ui/assets/35179038/78312d90-bf60-471d-90ab-4ae6fa94f07a">

Reply:
rewritten to appearance